src/Symfony/Component/Scheduler/RecurringMessage.php

```diff
@@ -51,6 +53,23 @@ public static function trigger(TriggerInterface $trigger, object $message): self
         return new self($trigger, $message);
     }
 
+    /**
+     * Unique identifier for this message's context.
+     */
+    public function getId(): string
+    {
+        if (isset($this->id)) {
+            return $this->id;
+        }
+
+        return $this->id = hash('crc32c', implode('', [
```

Comment:
I used this hash alg because it's nice and short (for possibly displaying in debug:schedule and using in a future `schedule:run` command). Not sure if we need to be concerned about collisions.

Reply:
> Not sure if we need to be concerned about collisions.

I meant in a mathematical sense - how probable is a collision?